charts/corda-dev-prereqs/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: corda-dev-prereqs
 description: A Helm chart for installing Corda 5 development pre-requisites (Kafka and PostgreSQL)
 
 type: application
-version: 0.2.0
```

Comment:
At a minimum we should bump the minor version but potentially this is a breaking change so we should consider creating a new major version of the chart.